_data/members.yml

```
@@ -163,19 +163,26 @@ research_engineers:
     
 medical_doctors:
 
-  - firstname: 'Loïc'
-    lastname: 'Boussel'
+  - firstname: 'Yves'
+    lastname: 'Berthezene'
     institution: 'HCL - PUPH'
     picture: 'icon_doctor.jpg'
     speciality: "Radiologist"
+    expertise: "Neuroimaging"
+
+  - firstname: 'Sara'
+    lastname: 'Boccalini'
+    institution: 'HCL - MCU-PH'
+    picture: 'icon_doctor.jpg'
```

I assume you just forgot to link to her photo, since that you did add an image for Sara Boccalini :wink: 
```suggestion
    picture: 'sBocallini.jpg'
```